content/articles/introduction-to-kernel-methods-in-machine-learning/fixed index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /introduction-to-kernel-methods-in-machine-learning/
+title: Introduction to Kernel Methods in Machine Learning 
+description: In this tutorial, we will learn about kernel methods in machine learning and how to use them to solve classification and regression problems.
+author: brian-kiplangat
+date: 2022-01-14T00:00:00-10:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/introduction-to-kernel-methods-in-machine-learning/hero.png
+    alt: Introduction to Kernel Methods in Machine Learning 
+---
+
+### Prerequisites
+- The Reader should have basic knowledge of machine learning. [Click here](https://www.digitalocean.com/community/tutorials/an-introduction-to-machine-learning) to read more about machine learning basics.
+- The reader should understand basic terms like Support Vector Machine, classifications, unsupervised learning, and supervised learning.
+- It's necessary for the reader to understand SVMs since kernels are implemented in SVM models. 
+### Introduction

```

Comment:
"Introduction" should come before the "Prerequisite"

Add a "Table of content" section.